miden-lib/asm/miden/contracts/faucets/basic_fungible.masm

```diff
@@ -33,8 +33,7 @@ const.METADATA_SLOT=1
 export.basic::auth_tx_rpo_falcon512
 
 #! Distributes freshly minted fungible assets to the provided recipient.
-#!
-#! Inputs: [amount, tag, RECIPIENT, ...]
+#! Inputs: [amount, tag, note_type, RECIPIENT]
```

Comment:
nit: I would also list `note_type` in the bullet point list below (e.g., on line 40).